### ISSUE TITLE
Incorrect animation when using Better Grass option of Optifine

### ISSUE THREAD

**Issue by bl4ckscor3** [TominoCZ/FancyBlockParticles#29]
If the Better Grass option of Optifine is turned on (Fast or Fancy) placing grass blocks looks as if only the top side of the block is being animated. [See this gfycat](https://gfycat.com/SparklingGreenAtlasmoth).

**Comment by TominoCZ** [comment]
hmm.. I looked into optifine's source code and all I could find is that optifine has it's own grass block model when that feature is on, messing it all up..

**Comment by bl4ckscor3** [comment]
I've tried all possible combinations between VBO on/off/50%, SL on/off, BG Fast/Fancy and the issue persist throughout them.

**Comment by TominoCZ** [comment]
yeah so that means that it completely changed the model of the block in a way that my mod can't disassemble it...

I'll look more into it..
